src/CONST.js

```diff
@@ -296,6 +297,7 @@ const CONST = {
     GOOGLE_MEET_URL_ANDROID: 'https://meet.google.com',
     DEEPLINK_BASE_URL: 'new-expensify://',
     PDF_VIEWER_URL: '/pdf/web/viewer.html',
+    CLOUDFRONT_DOMAIN_REGEX: new RegExp(`^https://\\w+\\.${CLOUDFRONT_DOMAIN.replace(/\./g, '\\.')}`, 'i'),
```

Comment:
What I meant here is to use `/regex format/i` (do not use RegExp for consistency). Please simply this and get rid of the replace method.